cartridge-cli.lua

```diff
@@ -1050,9 +1051,11 @@ local function get_rock_versions(project_dir)
     return dependencies
 end
 
-local function generate_version_file(source_dir, dest_dir, app_name, app_version)
+local function generate_version_file(distribution_dir)
+    info('Generate VERSION file')
+
     local version_file, _ = fio.open(
-        fio.pathjoin(dest_dir, 'VERSION'),
+        fio.pathjoin(distribution_dir, 'VERSION'),
```

Comment:
destination? or distribution?

Reply:
In this project name `distribution_dir` is used for directory where application files are placed (`/usr/share/tarantool/<appname>` contents).

I use `dest_dir` (destination directory) name to specify path where the result package should be placed.

Reply:
Okay